### ISSUE TITLE
Focus indicator in TitleBlockZen does not meet minimum contrast requirements

### ISSUE THREAD

**Issue by lloydstubber** [cultureamp/kaizen-design-system#1925]
TLDR; Increase the contrast on this component when focused

Type:WCAG 2
Principle: Perceivable
Guideline:1.4 Distinguishable
Success Criteria: 1.4.11 Non-text Contrast (AA)
Level: A
Platform: All
Severity:Medium

### What's the issue?
The focus indicator for the list items (e.g. Next: Aug 19, 2021) for the other actions button (...) does not meet minimum contrast requirements.
Focus indicator: #E6F6FF
Background: #FFFFFF
The focus indicator has a contrast ratio of 1.1:1 against the background.

<img width="603" alt="Screen Shot 2021-09-02 at 4 28 56 pm" src="https://user-images.githubusercontent.com/18339250/131793506-729212f7-2694-410b-8d8c-c35a521a8f97.png">

### Why it matters
People with low vision or colour blindness may have trouble keeping track of their position as they navigate the site using a keyboard.

### Our recommendation
Increase the contrast between the focus indicator and the background to be at least 3:1
Note: Other elements in the header use a border in addition to the background colour change.

**Comment by dougmacknz** [comment]
Closing this because I'm 99.9% sure now that this is due to dependencies being out of date - turns out the focus indicators were added recently in https://github.com/cultureamp/kaizen-design-system/pull/1781
